api/src/main/java/org/pmiops/workbench/api/UserMetricsController.java

```diff
@@ -121,17 +120,6 @@ public ResponseEntity<WorkspaceResource> updateRecentResource(
     return ResponseEntity.ok(toWorkspaceResource(recentResource, fcWorkspace, dbWorkspace));
   }
 
-  @Override
-  public ResponseEntity<EmptyResponse> deleteRecentResource(
```

Comment:
This function is not used. 
The delete function called here is based on the resource being deleted:
[First](https://github.com/all-of-us/workbench/blob/main/ui/src/app/components/resource-list.tsx#L134)

https://github.com/all-of-us/workbench/blob/eaf343ce79ae17c6362f18e65fa2cb52c30c92f8/ui/src/app/components/render-resource-card.tsx#L28

[Function called when delete is called](https://github.com/all-of-us/workbench/blob/main/ui/src/app/pages/analysis/notebook-action-menu.tsx#L158)